core/cli/node/client.hpp

```diff
@@ -0,0 +1,683 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <cinttypes>
+#include "api/full_node/node_api.hpp"
+#include "cli/node/node.hpp"
+#include "cli/validate/address.hpp"
+#include "cli/validate/cid.hpp"
+#include "codec/json/json.hpp"
+#include "common/cli_deal_stat.hpp"
+#include "common/enum.hpp"
+#include "common/table_writer.hpp"
+#include "markets/storage/mk_protocol.hpp"
+#include "primitives/atto_fil.hpp"
+#include "primitives/chain_epoch/chain_epoch.hpp"
+#include "primitives/piece/piece.hpp"
+#include "storage/car/car.hpp"
+#include "storage/ipfs/api_ipfs_datastore/api_ipfs_datastore.hpp"
+#include "storage/ipld/memory_indexed_car.hpp"
+#include "storage/unixfs/unixfs.hpp"
+#include "vm/actor/actor.hpp"
+#include "vm/actor/builtin/states/verified_registry/verified_registry_actor_state.hpp"
+#include "vm/actor/builtin/types/market/policy.hpp"
+#include "vm/actor/builtin/v0/verified_registry/verified_registry_actor.hpp"
+#include "vm/state/impl/state_tree_impl.hpp"
+
+namespace fc::cli::cli_node {
+  using api::FileRef;
+  using api::FullNodeApi;
+  using api::ImportRes;
+  using api::MarketBalance;
+  using api::QueryOffer;
+  using api::RetrievalOrder;
+  using api::StartDealParams;
+  using api::StorageMarketDealInfo;
+  using common::toString;
+  using ::fc::storage::unixfs::wrapFile;
+  using markets::retrieval::client::RetrievalDeal;
+  using markets::storage::DataRef;
+  using markets::storage::StorageDeal;
+  using markets::storage::StorageDealStatus;
+  using markets::storage::client::import_manager::Import;
+  using primitives::AttoFil;
+  using primitives::ChainEpoch;
+  using primitives::DealId;
+  using primitives::StoragePower;
+  using primitives::address::Address;
+  using primitives::piece::UnpaddedPieceSize;
+  using proofs::padPiece;
+  using storage::ipfs::ApiIpfsDatastore;
+  using vm::actor::Actor;
+  using vm::actor::kVerifiedRegistryAddress;
+  using vm::actor::builtin::states::VerifiedRegistryActorStatePtr;
+
+  static const ChainEpoch kLookBack = 100 * kEpochsInDay;
```

Comment:
Lookback is too big, for message that was just sent

Reply:
that is lotus notary messages waiting bound, what limit do you think will be more suitable than current one?

Reply:
100 is sufficient, i think

Reply:
Fixed